webrender/src/resource_cache.rs

```diff
@@ -675,13 +675,15 @@ impl ResourceCache {
         }
     }
 
-    fn update_texture_cache(cached_images: &mut ResourceClassCache<ImageRequest, CachedImageInfo>,
-                            texture_cache: &mut TextureCache,
-                            current_frame_id: FrameId,
+    fn update_texture_cache(&mut self,
```

Comment:
Here is another version of update_texture_cache(). This version spends one more "image_templates" hash table lookup, but it turns to a member function.
I still prefer the static function one.